src/savegame_json.cpp

```diff
@@ -1889,10 +1888,7 @@ void monster::store( JsonOut &json ) const
     }
     json.member( "inv", inv );
 
-    if( dragged_foe ) {
-        json.member( "dragged_foe_id", dragged_foe->getID() );
-    }
-
+    json.member( "dragged_foe_id", dragged_foe_id );
```

Comment:
You can save some storage space by not saving `dragged_foe_id` if it's invalid here, since monst monsters don't drag their opponents.

Reply:
I can, but we don't do so for most of the other attributes; most mobs aren't underwater or babies, for example. Having it always be written to the file makes the code easier to read and makes it easier to verify correct values are being stored by peeking at the save-files.

If we particularly cared about file storage then I'd hook into `json.member` itself, so it doesn't write values which are the same as the their defaults. But that's only something I'd evaluate once we start being concerned mobs are taking up too much space in the save files.